rousseau_vote/lib/src/widgets/poll_details_body_content.dart

```diff
@@ -55,9 +59,17 @@ class _PollDetailsBodyContentState extends State<PollDetailsBodyContent> {
           ),
         ),
         const SizedBox(height: 10),
+        _poll.optionType == 'ENTITY' ? 
```

Comment:
Are we assuming entities are people? If I remember correctly, this can be applied to different type of entities (like projects).